README.md

```diff
@@ -18,6 +18,11 @@ Configure `datadog_logs` plugin with your Datadog API key:
 output {
     datadog_logs {
         api_key => "<DATADOG_API_KEY>"
+        host => "intake.logs.datadoghq.com"
+        port => 10515
```

Comment:
should we use 10516?

Reply:
also, should we state that host and port default to those values? It is not obvious that we don't need to set them